pkg/odo/cli/pipelines/environment/add.go

```diff
@@ -28,14 +28,14 @@ var (
 )
 
 // AddEnvParameters encapsulates the parameters for the odo pipelines init command.
-type AddEnvParameters struct {
-	envName  string
-	output   string
-	manifest string
-	cluster  string
-	// generic context options common to all commands
-	*genericclioptions.Context
-}
+// type AddEnvParameters struct {
+// 	envName       string
+// 	output        string
+// 	pipelinesFile string
+// 	cluster       string
+// 	// generic context options common to all commands
+// 	*genericclioptions.Context
+// }
 
```

Comment:
AddEnvParameters should stay in add.go and should not be moved to environment.go.